app/controllers/admin/questions_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Admin::QuestionsController < AdminController
+  def index
+    @bank_questions = Question.for_banks
+    @partner_questions = Question.for_partners
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = Question.create(question_params)
+    if @question.valid?
+      redirect_to admin_questions_path
+    else
+      flash[:error] = "Failed to create question. #{error_message_for_answer(@question.errors)}"
+      render :new
+    end
+  end
+
+  def edit
+    @question = current_question
+  end
+
+  def update
+    @question = current_question
+    @question.update(question_params)
+    if @question.valid?
+      redirect_to admin_questions_path
+    else
+      flash[:error] = "Failed to update question. #{error_message_for_answer(@question.errors)}"
+      render :edit
+    end
+  end
+
+  def destroy
+    @question = current_question
+    @question.destroy
+    redirect_to admin_questions_path
+  end
+
+  private
+
+  def error_message_for_answer(errors)
```

Comment:
Why not use the contents of the `errors` itself instead of using a string? I think you can "humanize" the errors that you get out of the box.

Reply:
I refactored it the way that you mentioned. Please let me know if it needs any more changes.